contribute/index.md

```diff
@@ -41,36 +38,12 @@ templates, or directly from your terminal:
 root -b -e '.gh bug' -q
 root -b -e '.gh feature' -q
 root -b -e '.gh improvement' -q
-``
-
-## Picking up an idea
-
-We maintain a set of "ideas" for talented scientists and developers to pick up.
-An "idea" is a sketch of a development project, or a missing feature we would like to see in your ROOT!
-You can inspect the ideas in the following list.
-
-{% assign sorted = site.ideas | reverse %}
-
-### Some Ideas <a href="{{ 'feed/ideas.xml' | relative_url }}"><img style="width:auto; height:1.0em;" src="{{'/assets/images/feed.svg' | relative_url}}"></a>
+```
 
-<ul>
-{% for idea in sorted %}
-{% if idea.state != "completed" %}
-<li> {{idea.date | date_to_string }} <a href="{{ idea.url | relative_url }}"> {{ idea.title}} </a><br>
-{{idea.summary}}
-</li>
-{% endif %}
-{% endfor %}
-</ul>
+### What about fixing an "easy" bug?
 
-### Completed ideas
+Several of ROOT’s bugs are actually not all that difficult to fix - we just don’t get around
```

Comment:
...if you are a good C++ programmer that knows their way around a debugger and can handle large code bases :)

Reply:
Do we really want to add this sentence? it sounds a bit "elitist".  What about something more neutral:

_Several of ROOT’s bugs are actually not all that difficult to fix for a C++ programmer._

Reply:
yes that was a comment for you(s), not an edit suggestion :D 

To elaborate a bit: it's about managing expectations. we have seen many times that the "good first issue" label can mislead inexperienced developers in search of some "github street cred" into thinking that they can approach an issue, when it's not actually true, to their frustration as well as ours.

Just look at our list at https://github.com/root-project/root/labels/good%20first%20issue and think which of those could be picked up by the "mean square junior dev" without a lot of coaching. I think maybe 1 or 2.

Reply:
I totally agree. I will try to phrase it.